packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -112,4 +112,13 @@ Tab.propTypes = {
   announcerText: PropTypes.string
 };
 
+Tab.defaultProps = {
+  selected: false,
+  action: () => {},
```

Comment:
Should be `noop` for consistency.

Reply:
I was just using noop when it was already imported. I like this shorthand better since it means I don't have to add another import, but if we're valuing that consistency, I'm ok with it.